ast/edit/option_editor.go

```diff
@@ -0,0 +1,141 @@
+package edit
+
+import (
+	"fmt"
+
+	"github.com/influxdata/flux/ast"
+	"github.com/influxdata/flux/semantic"
+	"github.com/influxdata/flux/values"
+	"github.com/pkg/errors"
+)
+
+var OPT_NOT_FOUND = errors.New("option not found")
```

Comment:
While somewhat common in Go its best to avoid these "sentinel" errors as it creates a dependency on the package to be able to handle errors. See this blog post for some good discussion on the issue https://dave.cheney.net/2016/04/27/dont-just-check-errors-handle-them-gracefully.

Reply:
I read it and it made me conclude that I shouldn't return an error for that specific case